docs/getting-start.md

```diff
@@ -79,203 +79,207 @@ required for testing locally but is required for testing on your personal fork.
-        "ec2:DescribeVpcs",
-        "ec2:GetPasswordData",
-        "ec2:ImportKeyPair",
-        "ec2:ModifyInstanceAttribute",
-        "ec2:RevokeSecurityGroupEgress",
-        "ec2:RunInstances",
-        "ec2:RunInstances",
-        "ec2:TerminateInstances",
-        "ecr:BatchCheckLayerAvailability",
-        "ecr:BatchGetImage",
-        "ecr:CompleteLayerUpload",
-        "ecr:DescribeImages",
-        "ecr:DescribeRepositories",
-        "ecr:GetAuthorizationToken",
-        "ecr:InitiateLayerUpload",
-        "ecr:ListImages",
-        "ecr:ListTagsForResource",
-        "ecr:PutImage",
-        "ecr:UploadLayerPart",
-        "ecs:CreateCapacityProvider",
-        "ecs:CreateCluster",
-        "ecs:CreateService",
-        "ecs:CreateTaskSet",
-        "ecs:DeleteCapacityProvider",
-        "ecs:DeleteCluster",
-        "ecs:DeleteService",
-        "ecs:DeleteTaskSet",
-        "ecs:DeregisterTaskDefinition",
-        "ecs:DescribeCapacityProviders",
-        "ecs:DescribeClusters",
-        "ecs:DescribeContainerInstances",
-        "ecs:DescribeServices",
-        "ecs:DescribeTaskDefinition",
-        "ecs:DescribeTasks",
-        "ecs:ListClusters",
-        "ecs:ListContainerInstances",
-        "ecs:ListServices",
-        "ecs:ListTagsForResource",
-        "ecs:ListTaskDefinitions",
-        "ecs:ListTasks",
-        "ecs:PutClusterCapacityProviders",
-        "ecs:RegisterTaskDefinition",
-        "ecs:RunTask",
-        "ecs:StartTask",
-        "ecs:StopTask",
-        "ecs:UpdateService",
-        "iam:AddRoleToInstanceProfile",
-        "iam:AttachRolePolicy",
-        "iam:CreateInstanceProfile",
-        "iam:CreatePolicy",
-        "iam:CreateRole",
-        "iam:CreateServiceLinkedRole",
-        "iam:DeleteInstanceProfile",
-        "iam:DeletePolicy",
-        "iam:DeleteRole",
-        "iam:DetachRolePolicy",
-        "iam:GetInstanceProfile",
-        "iam:GetPolicy",
-        "iam:GetPolicyVersion",
-        "iam:GetRole",
-        "iam:ListAttachedRolePolicies",
-        "iam:ListInstanceProfilesForRole",
-        "iam:ListPolicyVersions",
-        "iam:ListRolePolicies",
-        "iam:PassRole",
-        "iam:RemoveRoleFromInstanceProfile",
-        "logs:CreateLogGroup",
-        "logs:DeleteLogGroup",
-        "logs:DescribeLogGroups",
-        "logs:ListTagsForResource",
-        "logs:ListTagsLogGroup",
-        "s3:DeleteObject",
-        "s3:GetObject",
-        "s3:GetObjectAcl",
-        "s3:ListBucket",
-        "s3:PutObject",
-        "ssm:DeleteParameter",
-        "ssm:DeleteParameters",
-        "ssm:DescribeParameters",
-        "ssm:GetParameter",
-        "ssm:GetParameters",
-        "ssm:ListTagsForResource",
-        "ssm:PutParameter",
-        "sts:GetCallerIdentity"
-      ],
-      "Resource": "*"
+      "Version": "2012-10-17",
+      "Statement": [
+        {
+          "Effect": "Allow",
+          "Action": [
+            "autoscaling:CreateAutoScalingGroup",
```

Comment:
Probably. I kind of rushed through the doc part of it. I will revisit this today

Reply:
I went through my CloudTrail and I must be doing something wrong because I can't see the EKS API calls that are being invoked. I think my role is just blanket access, I didn't attach anything specifically to get it to work so maybe I have to go through and redo that, or leave that mess for the next person. Idk

Reply:
That is odd.. when I was doing the setup on my fork, it did complain about things like "eks:CreateCluster", "eks:DescribeCluster" and "eks:CreateNodegroup"; after which I gave it blanket access as well :/
For someone that didnt grant access, it will clearly complain about what is missing - so Im fine either way.

Reply:
Am updating the doc to attach the AWS managed EKS cluster policy which should cover this and be more future proof than us manually listing permissions